src/Symfony/Component/Config/Resource/GlobResource.php

```diff
@@ -95,6 +99,16 @@ public function getIterator()
 
         if (0 !== strpos($this->prefix, 'phar://') && false === strpos($this->pattern, '/**/') && (\defined('GLOB_BRACE') || false === strpos($this->pattern, '{'))) {
             foreach (glob($this->prefix.$this->pattern, \defined('GLOB_BRACE') ? GLOB_BRACE : 0) as $path) {
+                if (is_dir($path)) {
+                    if ($this->forExclusion) {
+                        yield $path => new \SplFileInfo($path);
+                        continue;
+                    }
+
+                    if (array_key_exists(str_replace('\\', '/', $path), $this->excludedPrefixes)) {
```

Comment:
Is the str_replace needed for Windows? If yes, I'd suggest to use `str_replace(\DIRECTORY_SEPARATOR, '/', $path)` instead.

Reply:
Check @stof comment https://github.com/symfony/symfony/pull/28200#discussion_r209897523
So I just copied how it's done in https://github.com/symfony/dependency-injection/blob/master/Loader/FileLoader.php#L117

Do you still want me to chenge it?

Reply:
yes please, `\` is a legal character in a path on Linux (even though unusual of course)
that line could be changed also for sure

Reply:
Ok, should I also do another PR to change https://github.com/symfony/dependency-injection/blob/master/Loader/FileLoader.php#L117 too?

Reply:
if you're up to yes :) please check the full code base in case there are any other places that need such change (on branch 2.8 since it's the lowest maintained branch)